smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/ShapeValueGenerator.kt

```diff
@@ -160,11 +161,12 @@ class ShapeValueGenerator(
             }
             ShapeType.BLOB -> {
                 if (shape.hasTrait<StreamingTrait>()) {
-                    writer.writeInline(".stream(\$N(data: ", SmithyStreamsTypes.Core.BufferedStream)
-                    ".data(using: .utf8)!, isClosed: true))"
+                    writer.writeInline(".stream(\$N(data: \$N(", SmithyStreamsTypes.Core.BufferedStream, FoundationTypes.Data)
+                    ".utf8), isClosed: true))"
```

Comment:
A little code cleanup & converted to using Swift string's utf8 view to create data.